docs/commands.md

```diff
@@ -97,6 +97,10 @@ Available filters:
 |valor|`!raid moltres valor`| Tracks moltres raids where controlling team is valor|
 |harmony|`!raid venusaur harmony`| Tracks venusaur raids in uncontested gyms|
 
+###### !unraid
+
+`!unraid snorlax` Removes tracking of Snorlax raids. 
```

Comment:
add also the way to remove tracking per raid tier: "`!unraid level5` Removes tracking of tier 5 raids"